lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -567,177 +568,182 @@ def write(chunk)
         log_name = "projects/#{@project_id}/logs/#{log_name(
           tag, group_level_resource)}"
 
-        # Does the actual write to the cloud logging api.
+        # Pile up the requests in the REST format. The gRPC path will decompose
+        # them later.
+        requests_to_send << Google::Apis::LoggingV2::WriteLogEntriesRequest.new(
+          entries: entries,
+          log_name: log_name,
+          resource: group_level_resource,
+          labels: group_level_common_labels,
+          partial_success: @partial_success)
+      end
+      requests_to_send.each do |request|
         client = api_client
```

Comment:
You can now inline this directly into the call...

Reply:
I've folded `api_client` into the call.

BTW, I created b/71806931 to audit the way we use the api client (we are initiating a new one for every request).

Reply:
SG. FWIW, I was thinking of simply folding it into the `write_request(api_client, request)` call, but your way is better.

Reply:
Cool cool.